### ISSUE TITLE
Default new users to standard booking policies

### ISSUE THREAD

**Issue by joshdanielson** [loconomics/loconomics#611]
- [x] default all users to booking request and cancellation policy 1 (?)

**Comment by IagoSRL** [comment]
Changed to:
- CancellationPolicyID set to `3` "Flexible"
- InstantBooking set to `false` "Acceptance required"

Created upgrade script that will change any non-member users to this defaults. It will get a copy of that users before do the change and print the result (just in case we want to communicate something or anything).

I implemented this not only using default values at creation of a listing, but both at create and update process, the membership status of the user is checked in order to enforce default values when required; this means some REST API work is done for that requirement at #328, while is not reflected in the UI still (but non member users will see how any change to policies gets reverted).
